core/tauri-build/src/lib.rs

```diff
@@ -478,7 +478,7 @@ pub fn try_build(attributes: Attributes) -> Result<()> {
     if i == last {
       println!("cargo:rustc-env=TAURI_ANDROID_PACKAGE_NAME_APP_NAME={w}");
     } else {
-      android_package_prefix.push_str(w);
+      android_package_prefix.push_str(&w.replace(['_', '-'], "_1"));
```

Comment:
`TAURI_ANDROID_PACKAGE_NAME_APP_NAME` seemed to be replaced in https://github.com/tauri-apps/tauri/blob/762cf31a113f33cd524c8f6fa52ca47db8ea3705/core/tauri-macros/src/mobile.rs#L46-L51 before it calls `android_binding!`

Should we move that replacement to L479 as well? So that we can do it in one place.

Reply:
good idea, pushed